src/sentry/static/sentry/app/components/globalSdkUpdateAlert.tsx

```diff
@@ -168,8 +171,17 @@ class InnerGlobalSdkSuggestions extends React.Component<Props, State> {
 }
 
 const Content = styled('div')`
+  display: flex;
+  flex-wrap: wrap;
+
+  @media (min-width: ${p => p.theme.breakpoints[0]}) {
+    justify-content: space-between;
```

Comment:
I might prefer grid here, since otherwise we won't have a gap if the text gets close

Reply:
It's actually not bad, it just looks like this right before it wraps
<img width="945" alt="Screen Shot 2021-04-14 at 4 23 47 PM" src="https://user-images.githubusercontent.com/4830259/114792767-d0686300-9d3d-11eb-9552-b0db454af9ed.png">